public/css/frontend.css

```diff
@@ -10475,3 +10475,11 @@ a.text-dark:focus {
   max-height: 90px;
 }
 
+.profile-image {
```

Comment:
Same comment as above, this is directly modifying the compiled css. This will be deleted whenever recompiled with `npm run dev` or `npm run prod`